api/src/main/java/org/openmrs/Condition.java

```diff
@@ -340,34 +340,34 @@ public boolean equals(Object o) {
 			return false;
 		}
 		
-		Condition condition = (Condition) o;
+		Condition conditionObj = (Condition) o;
```

Comment:
what do you think about `conditionToBeChecked`

Reply:
I'll be better!